compiler/optimizer/UseDefInfo.cpp

```diff
@@ -583,7 +583,7 @@ void TR_UseDefInfo::findTrivialSymbolsToExclude(TR::Node *node, TR::TreeTop *tre
          if (aux._neverWrittenSymbols.get(symRefNum))
             {
             aux._neverWrittenSymbols.reset(symRefNum);
-            aux._nodesByGlobalIndex[node->getGlobalIndex()] = node;
+            // aux._nodesByGlobalIndex[node->getGlobalIndex()] = node;
```

Comment:
Please delete rather than commening

Reply:
The code that requires this data structure still exists, but similarly has been commented out. Is it preferable to remove this and the code that relies on it, or leave both?

Reply:
I don't think there is any value to commenting it out - if we need the previous code we can find it in the revision control system. If the delta is massive can talk again, but commenting code instead of removing it just leads to confusion since people are not sure why it was commented and whether there was an intention to restore it or not.

Reply:
The commit has been updated to remove the data structure and the code that relied on it.